custom/ewsghana/tests/test_webusers_sync.py

```diff
@@ -57,6 +57,7 @@ def test_create_webuser(self):
         self.assertEqual(TEST_DOMAIN, domain_name)
         self.assertEqual(UserRole.get_read_only_role_by_domain(TEST_DOMAIN)._id,
                          ewsghana_webuser.get_domain_membership(TEST_DOMAIN).role_id)
+        ewsghana_webuser.delete()
```

Comment:
I'm a little conflicted about cleanup like this in the test (rather than in some teardown construct). If one of the asserts fails this will not run, which I'm assuming will cause other tests to fail. One way to work around that is to use try/finally, but that makes the code a little messier. No action needed, really, just pointing it out.